app/templates/components/land-use-map.hbs

```diff
@@ -8,7 +8,7 @@
   {{/map.source}} --}}
 
   {{#with (await vectorSource) as |vSource|}}
-    {{#map.source sourceId='pluto' source=vSource as |source|}}
+    {{#map.source sourceId='pluto_landuse' source=vSource as |source|}}
```

Comment:
Oh, whoa! Was this wrong? Good catch. Thanks.

Reply:
it wasn't wrong, it just conflicted with the source ID of `pluto` coming in through `layers-api`.